cmd/bulk_load_cassandra/main.go

```diff
@@ -197,12 +193,10 @@ outer:
 			continue
 		}
 
-		batch.Query(string(scanner.Bytes()))
```

Comment:
There are some mistakes.

bulk_load_cassandra only writes the first record to Cassandra. In my case, the speed is 300x faster than the truth.